.circleci/config.yml

```diff
@@ -68,21 +70,18 @@ workflows:
     jobs:
       - build-node13
       - build-node12
-      - build-node10:
-          filters:
-            tags:
-              only: /^xml-for-vscode@[0-9]+(\.[0-9]+)*/
+      - build-node10
       - deploy-npm:
           filters:
             tags:
-              only: /^RELEASE/
+              only: /^RELEASE/ | /^xml-for-vscode@[0-9]+(\.[0-9]+)*/

```

Comment:
the `|` is part of the regexp 
- /^RELEASE|^xml-for-vscode@[0-9]+(\.[0-9]+)*/

I am not sure if capturing groups are needed but it may make it easier to read
- /(^RELEASE)|(^xml-for-vscode@[0-9]+(\.[0-9]+)*)/